src/components/board/TagsTabSidebard.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<div>
+		<ul class="labels">
+			<li v-for="label in labels" :key="label.id">
+				<template v-if="editingLabelId === label.id">
+					<input v-model="editingLabel.title"><input v-model="editingLabel.color">
+					<button class="icon-checkmark" @click="updateLabel(label)" :disabled="!editLabelObjValidated" 
+					v-tooltip="{content: 'title and color must be provided', 
+					show: !editLabelObjValidated, trigger: 'manual' }" />
+
+					<button v-tooltip="t('deck', 'Cancel')" class="icon-close" @click="editingLabelId = null" />
+				</template>
+				<template v-else>
+					<span :style="{ backgroundColor: `#${label.color}`, color: `#white` }" class="label-title">
+						<span v-if="label.title">{{ label.title }}</span><i v-if="!label.title"><br></i>
+					</span>
+					 <button v-tooltip="t('deck', 'Edit')" class="icon-rename" @click="clickEdit(label)" />
+					
+				
+					<button v-tooltip="t('deck', 'Delete')" class="icon-delete" @click="deleteLabel(label.id)" />
+
+				</template>
+			</li>
+			
+			<li v-if="addLabel">
+				<template>
+				<input v-model="addLabelObj.title"><input v-model="addLabelObj.color">
+				<button class="icon-checkmark" @click="clickAddLabel()" :disabled="!addLabelObjValidated" 
+					v-tooltip="{content: 'title and color must be provided', 
```

Comment:
Should also be translatable, as this is a duplicate string with the above, this should be moved in a data field

Reply:
what do you mean by "data field"?

Reply:
Sorry, data field is the wrong term. Data object ;) 
https://v1.vuejs.org/guide/instance.html#Properties-and-Methods